modules/sno-du-configuring-the-distributed-units.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: CONCEPT
+[id="sno-configuring-the-distributed-units_{context}"]
+= Configuring the distributed units (DUs)
+
+This section describes what a DU configuration looks like.
```

Comment:
Does this section tell you what a DU configuration looks like or tell you what you need to do? I read your sentence to indicate I don't have to do anything.   

```
This section describes how to configure the cluster for DU workloads.
After you have installed...
Apply the configurations in this section after installation...
```

Reply:
@imiller0 Anything we can do to clarify this intro?

Reply:
This document describes a set of configuration for an OCP cluster so that it meets the feature and performance requirements necessary for running a DU application. There are portions of this content that need to be applied at install time and other configuration which can be applied post-install. Hope that helps clarify....